scripts/segmentation/eval_segmentation.py

```diff
@@ -0,0 +1,172 @@
+import os
```

Comment:
Did you rename `test.py` to `eval_segmentation.py`?

Reply:
Could you add the quantization to the original `test.py`?
If it is not possible, please change keep the orginal `test.py` and create a seperate file. Do not deleate existing files.

Reply:
Thanks for your comments. Will keep original `test.py` and add support for quantized model in this script.